include/preprocess/analytic_center_linear_ineq.h

```diff
@@ -0,0 +1,125 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2024 Vissarion Fisikopoulos
+// Copyright (c) 2024 Apostolos Chalkis
+// Copyright (c) 2024 Elias Tsigaridas
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+
+#ifndef ANALYTIC_CENTER_H
+#define ANALYTIC_CENTER_H
+
+#include <tuple>
+
+#include "max_inscribed_ball.hpp"
+
+template <typename VT, typename NT>
+NT get_max_step(VT const& Ad, VT const& b_Ax)
+{
+    const int m = Ad.size();
+    NT max_element = std::numeric_limits<NT>::lowest(), max_element_temp;
+    for (int i = 0; i < m; i++) {
```

Comment:
I think we mostly use the style 
```
for 
{
  ....
}
```
in the library, would you mind switch to that style?

Reply:
Thanks, done